### ISSUE TITLE
[Bug]: [Bug][a11y] Permission type cannot be selected or changed on permission.site with Talkback

### ISSUE THREAD

**Issue by SoftVision-LorandJanos** [mozilla-mobile/fenix#27314]
### Steps to reproduce

1. Turn TalkBack (or the available screen reader) on.
2. Go to permission.site and tap on one of the items.
3. Switch to another item.

### Expected behaviour

3. The first item is selected.
4. The second item is selected.

### Actual behaviour

3. Sometimes not even the first item is selected.
4. The second item is not selected.

### Device name

Google Pixel 4

### Android version

Android 12

### Firefox release type

Firefox Nightly

### Firefox version

107.0a1

### Additional information

https://user-images.githubusercontent.com/48829350/194392581-c9ae4623-98dd-4a84-9ecd-efe2245e5d7a.mp4

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21749)

**Comment by Mugurell** [comment]
I can reproduce on a OnePlus 7T with Android 11 and Talkback 13.0 while on Chrome there is no issue.

It seems to me that the green border does not appear when tapping on an element but will appear when navigating between elements.
In all cases a double click does work.

https://user-images.githubusercontent.com/11428869/194506983-507d2745-849a-4909-bd06-9e8248f48c10.mp4

@SoftVision-LorandJanos Can you confirm?

**Comment by SoftVision-LorandJanos** [comment]
@Mugurell for some elements, while the green border does appear when tapping between elements, it doesn't stay on.
And, yes, "double tap to activate" is announced twice like on #27311.

**Comment by Mugurell** [comment]
Since this issue happens in browser I'll move the ticket to Bugzilla for more investigation.

**Comment by Mugurell** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1794392

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
